src/routes/genreoverviewpage.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { GenreButton } from "../components/genre";
+import { genres, Genre } from "../utils/genre";
+import { Button } from "../components/button";
+import { IoIosArrowBack } from "react-icons/io";
+
+export function GenreOverviewPage() {
+  const [selectedGenres, setSelectedGenres] = useState<Genre[]>([]);
+  const navigate = useNavigate();
```

Comment:
Not the way to go, I believe. Causes an error for me. Please remove. 
No navigation needed for now!